README.md

```diff
@@ -1,67 +1,248 @@
-[Portfolio Website] is a designed to showcase my work and let others know about myself. 
+<a name="readme-top"></a>
 
-🛠 Built 
-HTML
-CSS
+<!--
+HOW TO USE:
+This is an example of how you may give instructions on setting up your project locally.
 
-Tech Stack
+Modify this file to match your project and remove sections that don't apply.
 
-Client
-[A html file with header section and headline section].
-[A css file that styles that styled both section and add background image to the page].
-[An md file required to be read].
+REQUIRED SECTIONS:
+- Table of Contents
+- About the Project
+  - Built With
+  - Live Demo
+- Getting Started
+- Authors
+- Future Features
+- Contributing
+- Show your support
+- Acknowledgements
+- License
 
+After you're finished please remove all the comments and instructions!
+-->
+
+<div align="center">
+  <!-- You are encouraged to replace this logo with your own! Otherwise you can also remove it. -->
+  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
+
+  <h3><b>Microverse README Template</b></h3>
```

Comment:
- [ ] [OPTIONAL] Please remove this heading and use your own project's title or repo name, as it will be more professional and clear for anyone who checks out our project, also this will personalise the readme for your own project.